mlir/include/mlir/Dialect/Arith/IR/ArithBase.td

```diff
@@ -156,4 +156,26 @@ def Arith_IntegerOverflowAttr :
   let assemblyFormat = "`<` $value `>`";
 }
 
+//===----------------------------------------------------------------------===//
+// Arith_RoundingMode
+//===----------------------------------------------------------------------===//
+
```

Comment:
Are these modes and their names defined somewhere that we can link here?

Reply:
These correspond to LLVM's. I can link.